packages/client/components/PalettePicker/PalettePicker.tsx

```diff
@@ -0,0 +1,90 @@
+import React, {useEffect, useState} from 'react'
+import {TemplatePromptItem_prompt} from '../../__generated__/TemplatePromptItem_prompt.graphql'
+import PaletteColor from '../PaletteColor/PaletteColor'
+import Menu from '../Menu'
+import {MenuProps} from '../../hooks/useMenu'
+import withAtmosphere, {WithAtmosphereProps} from '../../decorators/withAtmosphere/withAtmosphere'
+import withMutationProps, {WithMutationProps} from '../../utils/relay/withMutationProps'
+import ReflectTemplatePromptUpdateGroupColorMutation from '../../mutations/ReflectTemplatePromptUpdateGroupColorMutation'
+import styled from '@emotion/styled'
+import {PALETTE} from '../../styles/paletteV2'
+import {palettePickerOptions} from '../../styles/palettePickerOptions'
+
+interface Props extends WithAtmosphereProps, WithMutationProps {
+  prompt: TemplatePromptItem_prompt
+  prompts: any
+  menuProps: MenuProps
+}
+
+const PaletteDropDown = styled(Menu)({
+  width: '214px',
+  minWidth: '214px',
+  padding: 5
+})
+
+const PaletteList = styled('ul')({
+  display: 'flex',
+  flexWrap: 'wrap',
+  justifyContent: 'center',
+  listStyle: 'none',
+  padding: 0,
+  margin: 0
+})
+
+const PaletteItem = styled('div')({
+  cursor: 'grab'
+})
+
+const PalettePicker = (props: Props) => {
+  const {prompt, prompts, menuProps, atmosphere, onError, onCompleted, submitMutation} = props
+  const [groupColor, setGroupColor] = useState(prompt.groupColor)
+  const pickedColors = prompts.map((prompt) => prompt.groupColor) as string[]
+  const availableColors = palettePickerOptions.filter(
+    (color) => !pickedColors.includes(color)
+  ) as string[]
+
+  const handleClick = (color: string) => {
+    setGroupColor(color)
+    menuProps.closePortal()
+    ;(document as any).activeElement?.blur()
```

Comment:
I was referencing this idea from the `DueDatePicker` component but looking at it now, I think a better approach would be to avoid casting document entirely and just check if it has access to `HTMLElement` properties e.g.,
```
if (document.activeElement instanceof HTMLElement) {
    document.activeElement.blur();
}
```
The blur is just so the palette loses focus on close.